Pod/Classes/WPInputMediaPickerView.m

```diff
@@ -0,0 +1,87 @@
+#import "WPInputMediaPickerView.h"
+#import "WPPHAssetDataSource.h"
+
+@interface WPInputMediaPickerView()
+
+@property (nonatomic, strong) WPMediaPickerViewController *mediaPicker;
+@property (nonatomic, strong) UIToolbar *mediaToolbar;
+
+@end
+
+@implementation WPInputMediaPickerView
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self) {
+        [self commonInit];
+    }
+    return self;
+}
+
+- (instancetype)initWithCoder:(NSCoder *)aDecoder {
+    self = [super initWithCoder: aDecoder];
+    if (self) {
+        [self commonInit];
+    }
+    return self;
+}
+
+- (void)commonInit {
+    WPMediaPickerViewController *vc = [[WPMediaPickerViewController alloc] init];
```

Comment:
Containing a view controller inside of a view violates the containment policy of view controllers (addChild, etc). It seems like rotation works properly but there's a lot that goes on under the hood with VCs you're not getting doing it this way.